rules/windows/process_creation/proc_creation_win_susp_explorer_break_proctree.yml

```diff
@@ -1,25 +1,25 @@
-title: Explorer Root Flag Process Tree Break
+title: Explorer Process Tree Break
 id: 949f1ffb-6e85-4f00-ae1e-c3c5b190d605
 status: test
-description: Detects a command line process that uses explorer.exe /root, which is similar to cmd.exe /c, only it breaks the process tree and makes its parent a new instance of explorer
-author: Florian Roth
+description: Detects a command line process that uses explorer.exe to launch arbitrary commands or binaries, which is similar to cmd.exe /c, only it breaks the process tree and makes its parent a new instance of explorer spawning from "svchost"
+author: Florian Roth, Nasreddine Bencherchali
 references:
-  - https://twitter.com/CyberRaiju/status/1273597319322058752
-  - https://twitter.com/bohops/status/1276357235954909188?s=12
+    - https://twitter.com/CyberRaiju/status/1273597319322058752
+    - https://twitter.com/bohops/status/1276357235954909188?s=12
+    - https://twitter.com/nas_bench/status/1535322450858233858
```

Comment:
`- https://securityboulevard.com/2019/09/deobfuscating-ostap-trickbots-34000-line-javascript-downloader/`
not find a better ref to explain